encoding/masking-limiting-etc.md

```diff
@@ -33,8 +52,47 @@ and one of these formats:
 `GRAY16` (16 bits integer),
 or `GRAYS` (single precision floating point).
 
-Vapoursynth includes some basic tools for manipulating masks as well:
+#### [std.MaskedMerge][]
```

Comment:
Eh, going from from h1 to h4 is suboptimal, but using h2 for a filter seems superfluous.

Reply:
So what do you suggest, then? 🤔

Reply:
If I had a suggestion, I would have already given it. 😄 

I say leave it as h4 for now, because it's like that for the rest of the file already and looks good enough. Until we figure out something better, this will most certainly do. h3 is still too prominent in the default style.